pkgs/development/python-modules/wsgitools/default.nix

```diff
@@ -0,0 +1,30 @@
+{lib
+,buildPythonPackage
+,fetchgit
+}:
+
+buildPythonPackage rec {
+  pname = "wsgitools";
+  rev = "44bdfe2a595ce06dfda3d00ee0cf65e565c3cbea";
+  version = "0.3postgit20190310";
+
+  src = fetchgit {
+    inherit rev;
+    url = "git://git.subdivi.de/~helmut/wsgitools.git";
+    sha256 = "17i939zlk5k1hai51qyhm234fi7f0xc83qgh14r396khg9zsjjnw";
+  };
```

Comment:
can they do a new release on pypi? https://pypi.org/project/wsgitools/ seems 5 years out of date anyway

Reply:
For http2tcp there is already a new release, for wsgitools I have asked for one.

Reply:
For http2tcp there is already a new release, for wsgitools I have asked for one.

Reply:
Now wsgitools comes from pypi at 0.3.1